.markdownlint.json

```diff
@@ -0,0 +1,11 @@
+{
+    "default": true,
+    "MD013": {
+        "code_block_line_length": 999,
+        "tables": false
+    },
+    "MD014": false,
+    "MD025": false,
+    "MD026": false,
+    "MD033": false
+}
```

Comment:
Leaving a breadcrumb of https://github.com/DavidAnson/markdownlint#rules--aliases for anyone who's looking at this and wondering what rules we're using.